gbpservice/nfp/lib/transport.py

```diff
@@ -174,27 +174,43 @@ def send_request_to_configurator(conf, context, body,
     # method (tcp_rest/rpc) for sending request to configurator.
     if device_config:
         method_name = method_type.lower() + '_network_function_device_config'
-        for ele in body['config']:
-            ele['kwargs'].update({'context': context.to_dict()})
+        body['info']['context'].update({'neutron_context': context.to_dict()})
     elif network_function_event:
         method_name = 'network_function_event'
     else:
-        if (body['config'][0]['resource'] in ['ansible', 'heat']):
-            body['config'][0]['kwargs'].update({'context': context.to_dict()})
+        if (body['config'][0]['resource'] in ['heat', 'ansible']):
```

Comment:
We need to send the neutron_context for *aaS also right? Is that code not part of this file?